js/ui/codeView.js

```diff
@@ -843,7 +847,7 @@ var CodingSession = new Lang.Class({
         // If we have an app window, disconnect any signals and destroy it,
         // unless we are destroying the session because the app window was
         // destroyed in the first place
-        if (this.app) {
+        if (this.app && !this.app.is_destroyed()) {
```

Comment:
Not sure if this change is required anymore now that we avoid having a second handleDestroyWindow callback running?

Reply:
This is needed and I've to add the same to the toolbox  because if we don't do this, when we close when the app is minimized, this function is called with the app/toolbox destroyed and the `meta_window` will be null and the following code will fail.